pkg/bpm/bpm.go

```diff
@@ -230,24 +233,21 @@ type ProcessBuilder struct {
 	ctx context.Context
 }
 
-// SetNetNS sets the net namespace of the process
-func (b *ProcessBuilder) SetNetNS(nsPath string) *ProcessBuilder {
-	return b.SetNS([]nsOption{{
-		Typ:  NetNS,
-		Path: nsPath,
-	}})
+// GetNsPath returns corresponding namespace path
+func GetNsPath(pid uint32, typ NsType) string {
+	return fmt.Sprintf("%s/%d/ns/%s", DefaultProcPrefix, pid, string(typ))
 }
 
-// SetPidNS sets the pid namespace of the process
-func (b *ProcessBuilder) SetPidNS(nsPath string) *ProcessBuilder {
-	return b.SetNS([]nsOption{{
-		Typ:  PidNS,
-		Path: nsPath,
+// SetNS sets the namespace of the process
+func (b *ProcessBuilder) SetNS(pid uint32, typ NsType) *ProcessBuilder {
```

Comment:
we use `GetNsPath` to get the path, and then use the path as an argument in `SetNetNS` and `SetPidNS`, and we can use `SetNS` directly.